lib/registry.js

```diff
@@ -65,6 +65,7 @@ exports.load = function(registry) {
     }, globalConfig.config.registries[registry] || {});
 
     options.name = registry;
+    options.strictSSL = 'strictSSL' in globalConfig.config ? globalConfig.config.strictSSL : true;
```

Comment:
Surely this can just be `globalConfig.config.strictSSL` if we have the default set in global-config as above?